### DISCUSSION TITLE
Imported Material icons looks to big on TabBar

### DISCUSSION THREAD

**Discussion by Sinan-Karakaya** [skiptools/skip#278] [Q&A]
When importing Material Icons following the [documentation](https://skip.tools/docs/modules/skip-ui/#system-symbols), and using them for a tab bar like so, I get the following results:

```swift
NavigationStack {
                
}
.tabItem { Label("New", systemImage: "plus.app.fill") }
.tag(Tab.new)
```

<img width="1191" alt="image" src="https://github.com/user-attachments/assets/cf492c43-36ed-479d-ab16-028eebdef6fd">

<img width="183" alt="image" src="https://github.com/user-attachments/assets/383f722d-a649-47dd-81c5-7e1f1cfe27e1">

The image on the right appear clearly bigger. I used a tool to make the Material svg conform to what XCode wants for a symbol image set. [(This one)](https://wangchujiang.com/create-custom-symbols/).

I am not sure what is causing this, I already tried lowering the svg size from 24 to 20, but this didn't change anything.

**Comment by ameyp** [comment]
I'm experiencing the same behavior. I went to https://fonts.google.com/icons?selected=Material+Symbols+Outlined:stop:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=stop&icon.size=24&icon.color=%23e3e3e3&icon.platform=ios, then the "Apple" tab in the right sidebar and clicked on download. That gave me an SVG that I imported into my project.

![Screenshot 2025-05-16 at 7 38 52 AM](https://github.com/user-attachments/assets/0ae3cbde-8897-46ac-9102-47f71aee965d)

The result looks way bigger than the (builtin?) play symbol on Android and the builtin play/stop symbols on iOS.

![Screenshot 2025-05-16 at 7 41 28 AM](https://github.com/user-attachments/assets/abf9f48a-652e-42dd-a556-66a476e6db3f)

![Screenshot 2025-05-16 at 7 41 37 AM](https://github.com/user-attachments/assets/1b535229-fa69-447d-bf56-818b08f32ae5)
